.circleci/config.yml

```diff
@@ -127,6 +127,60 @@ jobs:
       - store_artifacts:
           path: junit
 
+  lint_eslint:
+    docker:
+      - image: circleci/node:6.10.3
+    working_directory: ~/bit
+    steps:
+      - run:
+          name: save SHA to a file
+          command: echo $CIRCLE_SHA1 > .circle-sha
+      - restore_cache:
+            key: builded-{{ checksum ".circle-sha" }}
```

Comment:
why are you restore the builded code instead of the source code? there is a special step for checkout code